src/app/header/header.component.scss

```diff
@@ -1,107 +1,132 @@
 @import '../../assets/stylesheets/base';
 
-.dropdown {
-    a:hover {
-        text-decoration: none;
-    }
-    a {
-        color: white;
-        background-color: transparent !important;
-    }
-}
+$pf-cust-navbar-text-size:$font-size-large;
+
+$pf-custom-navbar-top-border-width:2px;
+$pf-custom-navbar-header-border-width:0px;
```

Comment:
Did we agree to no units on zero?
If we did, then there are at least 2 of them with units here.